RetakesAllocator/Menus/AdvancedGunMenu.cs

```diff
@@ -0,0 +1,617 @@
+using CounterStrikeSharp.API.Core;
+using CounterStrikeSharp.API.Modules.Entities.Constants;
+using CounterStrikeSharp.API.Modules.Utils;
+using RetakesAllocatorCore;
+using CounterStrikeSharp.API;
+using System.Text;
+using RetakesAllocator.Menus;
+using RetakesAllocatorCore.Config;
+using static RetakesAllocatorCore.PluginInfo;
+
+namespace RetakesAllocator.AdvancedMenus;
+
+public class AdvancedGunMenu
+{
+    private RetakesAllocator retakesAllocatorInstance;
+    public AdvancedGunMenu(RetakesAllocator allocatorInstance)
+    {
+        this.retakesAllocatorInstance = allocatorInstance;

```

Comment:
Doesnt seem like this `allocatorInstance` is actually used, can we remove it

Reply:
blame github on that i dont it manual update to not get alot of Commits all of reds and greens x)